### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Luna-meili**
Ready for review, undated with information about Hacktoberfest contributos guidelines @CaroFG @ferdi05

# Pull Request

## What does this PR do?
Updating the contributing file to include Hacktoberfest 2022

## PR checklist
Please check if your PR fulfills the following requirements:
- [ ] Does this PR fix an existing issue?
- [ ] Have you read the contributing guidelines?
- [ ] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Your contributions are always welcome!
 
+<h3>It&rsquo;s Hacktoberfest 2022 @Meilisearch</h3>
+<p><a href="https://hacktoberfest.com">Hacktoberfest</a> is a celebration of the open-source community. This year, and for the third time in a row, Meilisearch is participating in this fantastic event.</p>
+<p>You&rsquo;d like to contribute? Don&rsquo;t hesitate to check out our <a href="https://github.com/meilisearch/meilisearch-ruby/blob/main/CONTRIBUTING.md">contributing guidelines</a>.<br /><br />And the longer, more complete guideline documentation you can follow <a href="https://github.com/meilisearch/.github/blob/main/Hacktoberfest_2022_contributors_guidelines.md">this link</a>.</p>
```
Comment (ferdi05) [inline CONTRIBUTING.md:7]: it's a link to the Ruby integration repo

**Review by ferdi05 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (CaroFG) [reply CONTRIBUTING.md:7]: I also would put this section in the README since it's the first think a person sees...

Comment (Luna-meili) [reply CONTRIBUTING.md:7]: I'll move it to the readMe and change the link!

Comment (ferdi05) [reply CONTRIBUTING.md:7]: Could you update your PR accordingly? thanks

**Review by ferdi05 — CHANGES_REQUESTED** [review]

**Comment by Luna-meili** [comment]
Contribution file to be left unchanged, the ReadMe will be updated accordingly

**Pull request closed without merging**